packages/amplify-graphql-transformer-core/src/cdk-compat/searchable-datasource.ts

```diff
@@ -0,0 +1,31 @@
+import { BackedDataSourceProps, BaseDataSource } from '@aws-cdk/aws-appsync';
+import { Construct } from '@aws-cdk/core';
+
+/**
+ * Properties for an AppSync searchable data source
+ */
+export interface SearchableDataSourceProps extends BackedDataSourceProps {
+  /**
+   * Region for the Amazon OpenSearch Service domain
+   */
+  readonly region: string;
+  /**
+   * Endpoint for the Amazon OpenSearch Service domain
+   */
+  readonly endpoint: string;
+}
+
+/**
+ * An AppSync data source backed by OpenSearch
+ */
+export class SearchableDataSource extends BaseDataSource {
+  constructor(scope: Construct, id: string, props: SearchableDataSourceProps) {
+    super(scope, id, props, {
+      type: 'AMAZON_OPENSEARCH',
```

Comment:
`type` went from `AMAZON_ELASTICSEARCH` to `AMAZON_OPENSEARCH`. Since `BaseDataSource` comes from a different package, I'm not sure if this will break something.